lib/elixir/lib/io.ex

```diff
@@ -301,37 +311,44 @@ defmodule IO do
 
   """
   @spec stream(device, :line | pos_integer) :: Enumerable.t
-  def stream(device, line_or_codepoints) do
+  def stream(device, line_or_codepoints)
+  when line_or_codepoints == :line
```

Comment:
In Elixir, `when` is usually indented two levels (4 spaces) more than the thing it belongs to (`def` here).